kubernetes/loculus/templates/ingest-deployment.yaml

```
@@ -24,6 +26,26 @@ spec:
         argocd.argoproj.io/sync-options: Force=true,Replace=true
```

I think this where we need the annotation:
```suggestion
        argocd.argoproj.io/sync-options: Force=true,Replace=true
        reloader.stakater.com/auto: "true"
```

---

Or no sorry this is a top level annotation so it should be in the metadata section not the metadata.template section. So maybe we should move both annotations up actually?

---

Indeed!